comma/individual.py

```diff
@@ -66,16 +76,76 @@ def choose_actions_on_lockdown(self, lockdown: pd.DataFrame)\
         n_actions, _ = params_lockdown.shape
         action_probs = params_lockdown.dot(self.get_features())
         # apply the sigmoid function
-        action_probs = np.asarray(
-            action_probs.apply(lambda x: 1 / (1 + np.exp(-x)))
-        )
-        actions = np.random.rand(n_actions) <= action_probs
+        action_probs = np.asarray(action_probs.apply(lambda x: 1 / (1 + np.exp(-x))))
+        # use the new random generator method of numpy
+        if rng is None:
+            rng = np.random.default_rng(12345)
+        actions = rng.random(n_actions) <= action_probs
         self.chosen_actions = actions  # store the chosen action
 
         return actions, action_probs
 
-    def take_actions(self, actions: pd.Series,
-                     action_effects: pd.DataFrame) -> None:
+    @staticmethod
+    def is_recovered(n_days, rng=None):
+        """
+        Determine if an individual is recovered based on
+        the number of days since testing positive.
+
+        The function uses a gamma cumulative distribution
+        function with shape parameter 5 and scale parameter 3
+        to model the probability of recovery. This can be of
+        course changed based on the literature.
+        Note that if `n_days` is equal or lower than 10,
+        then the probability of recovering is always 0,
+        as per Astrid's suggestion.
+
+        Args:
+            n_days (int): Number of days since tested positive.
+            rng (np.random.Generator): optional. An instance of numpy random
+            generator. If not provided, a default random generator will be
+            used. This ensures reproducibility.
+        Returns:
+            recovery (bool): True if recovered, False otherwise.
+        """
+
+        if n_days <= 10:
+            recovery = 0
+        else:
+            recovery_prob = gamma.cdf(n_days, a=5, scale=3)
+            # use the new generator method of numpy
+            if rng is None:
+                rng = np.random.default_rng(12345)
```

Comment:
`12345` is the random seed, correct? If so, I would store it globally, e.g. in `settings.py` so that it can easily be changed. That would make it easier to re-run simulations with varying seeds (including `None`).
```suggestion
                rng = np.random.default_rng(RANDOM_SEED)
```